### DISCUSSION TITLE
i need a jwt-guide ?

### DISCUSSION THREAD

**Discussion by nguyenphucminh** [nguyenphucminh/react_hook#4] [Q&A] [answered]
how to use jwt ??

**Comment by balebomm** [comment, marked as answer]
JSON Web Token (JWT) is an open standard ([RFC 7519](https://tools.ietf.org/html/rfc7519)) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object. This information can be verified and trusted because it is digitally signed. JWTs can be signed using a secret (with the HMAC algorithm) or a public/private key pair using RSA or ECDSA.
